presto-main/src/main/java/com/facebook/presto/execution/scheduler/FixedSourcePartitionedScheduler.java

```diff
@@ -156,9 +158,10 @@ public ScheduleResult schedule()
         // schedule a task on every node in the distribution
         List<RemoteTask> newTasks = ImmutableList.of();
         if (!scheduledTasks) {
-            OptionalInt totalPartitions = OptionalInt.of(partitioning.getPartitionToNode().size());
-            newTasks = partitioning.getPartitionToNode().entrySet().stream()
-                    .map(entry -> stage.scheduleTask(entry.getValue(), entry.getKey(), totalPartitions))
+            OptionalInt totalPartitions = OptionalInt.of(stageNodeList.size());
```

Comment:
Isn't `totalPartitions` the number of nodes? The term `partition` is overloaded and can confuse readers like me.

Reply:
@nezihyigitbasi : Yes. partition to node is 1 to 1 mapping. I think we might want to seek for some simplifications. But I suggest not to do it in this PR.